cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,21 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+    cy.visit('https://react-mobx.realworld.io/#/register');
+
+    cy.get('[placeholder=Username]')
+      .type(user.username);
+
+    cy.get('[placeholder=Email]')
+      .type(user.email);
+
+    cy.get('[placeholder=Password]')
+      .type(user.password);
+
+    cy.get('.btn').click();
 
+    cy.get(':nth-child(4) > .nav-link').should('contain', user.username);
```

Comment:
This is a bit cryptic and could use a comment explaining what it's checking for. It's also a good practice to add assertions for negative cases. For example, you could check that an error message does not exist after successful registration.